examples/stable_diffusion_v2/image2image.py

```diff
@@ -120,16 +121,24 @@ def main(args):
         apply_canny = CannyDetector()
         detected_map = apply_canny(img, low_threshold, high_threshold)
         detected_map = HWC3(detected_map)
-        Image.fromarray(detected_map).save(
-            os.path.join(outpath, f"{datetime.datetime.now().strftime('%Y%m%d-%H%M%S')}_detected_map.png")
-        )
-
-        control = ms.Tensor(detected_map.copy()).float() / 255.0
-        control = control.permute(2, 0, 1)
-        control = ops.stack([control for _ in range(num_samples)], axis=0)
+    elif args.mode == MODE["segmentation"]:
+        if os.path.exists(args.condition_ckpt_path):
+            apply_segment = SegmentDetector(ckpt_path=args.condition_ckpt_path)
+        else:
+            logger.warning(f"!!!Warning!!!: {args.condition_ckpt_path} doesn't exist")
```

Comment:
```suggestion
            logger.warning(f"Segmentation Detector checkpoint {args.condition_ckpt_path} was not found.")
```
Please use more descriptive warnings for easier debugging.

Reply:
fixed